packages/next/cli/next-dev.ts

```diff
@@ -220,9 +213,10 @@ const nextDev: cliCommand = async (argv) => {
           if (
             configKey === 'serverComponentsExternalPackages' ||
             configKey === 'appDir' ||
-            configKey === 'transpilePackages' ||
+            configKey === 'images' ||
```

Comment:
This looks wrong, probably needs a separate line?

Reply:
`transpilePackages` is not supported afaik, not sure why it was here

Reply:
We added it initially as it was mentioned that turbopack just transpiled everything so it kinda sorta supported the config